crates/iota-data-ingestion-core/src/reader.rs

```diff
@@ -48,13 +48,26 @@ pub struct CheckpointReader {
     data_limiter: DataLimiter,
 }
 
+/// Options for configuring how the checkpoint reader fetches new checkpoints.
 #[derive(Clone)]
 pub struct ReaderOptions {
+    /// - How often to check for new checkpoints.
+    /// - Lower values mean faster detection but more CPU usage.
+    /// - Default: 100ms.
```

Comment:
This will be rendered as an unordered list.

Reply:
refactored as ordered list, fixed: https://github.com/iotaledger/iota/pull/5735/commits/ba89ca54a65504e7e800ea02cdf439318cac99c2#diff-8b51598e4c5ff46aed5be1353cc7b783ffbdcbe6a1a3f8350b33742209a3710dR54-R70

Reply:
What I wanted to emphasize is that it will be rendered to a list as opposed to free text describing the field. It strikes me odd that we use a list for this purpose.

But if you must use a list, then unordered would probably be less of a surprise to the reader.

Apologies for not having been clearer in the first place.

Reply:
Fixed: https://github.com/iotaledger/iota/pull/5735/commits/fa7259e9da0c7cb678bb317213cefeb393df00bc.